sources/functions/utils

```diff
@@ -174,19 +174,23 @@ function check_ram() {
 # Enables a temporary 2GB swap file
 # \
 # Exits if cannot continue
-function tmp_swap_on() {
-    echo_progress_start "Setup is enabling the use of a temporary 2GB swap file"
-    dd if=/dev/zero of=/.swapfile bs=1M count=2048 > /dev/null 2>&1 || {
-        echo_error "Cannot continue with install because swap file could not be created. Does device have 2GB of free space?"
-        exit 1
-    }
-    mkswap /.swapfile > /dev/null 2>&1
-    swapon /.swapfile > /dev/null 2>&1
-    echo_progress_done 'Swap enabled'
+function check_swap_on() {
+    if ! check_ram 2G; then
+        echo_progress_start "Setup is enabling the use of a temporary 2GB swap file"
+        dd if=/dev/zero of=/.swapfile bs=1M count=2048 > /dev/null 2>&1 || {
```

Comment:
Do we want to check if this is available to begin with? if <2GB RAM is the case, chance is the device is running on like minimal storage mounted on root too